dashboards/service_test.go

```diff
@@ -21,7 +17,7 @@ func TestBoltDashboardService(t *testing.T) {
 }
 
 func initBoltDashboardService(f dashboardtesting.DashboardFields, t *testing.T) (influxdb.DashboardService, string, func()) {
-	s, closeBolt, err := newTestBoltStore(t)
+	s, closeBolt, err := itesting.NewTestBoltStore(t)
```

Comment:
Do we ever do anything with the returned error other than immediately fail the test? Maybe worth just putting a `t.Fatalf(err)` inside `NewTestBoltStore` and friends and avoid returning the error?

Reply:
Done